### ISSUE TITLE
ember-getowner-polyfill.import deprecation

### ISSUE THREAD

**Issue by thomblake** [ember-animation/liquid-fire#534]
I believe this starts happening after Ember 2.9 (can't find the actual deprecation link)

ember-getowner-polyfill is now a true polyfill. Use Ember.getOwner directly instead of importing from ember-getowner-polyfill [deprecation id: ember-getowner-polyfill.import]

**Comment by ef4** [comment]
Are you on liquid-fire 0.27.0? It should have addressed this deprecation warning.

**Comment by thomblake** [comment]
Yes, this is after upgrading. I'll take a look at where this is happening.

**Comment by craigsketchley** [comment]
I got this issue too. It seems to occur after https://github.com/rwjblue/ember-getowner-polyfill was updated (v1.0.1 works for me, but v1.1.x does not).

**Comment by msalahz** [comment]
I get the same , what is the status of this ?

**Comment by Vinchens00** [comment]
Hi guys,
I ran into the same issue. Do you have any updates here?

**Comment by ef4** [comment]
There is a bugfix in 0.27.1, does it fix this use case?

**Comment by Vinchens00** [comment]
It doesn't for me.

**Comment by serv** [comment]
Still happening on Ember 2.12.1

**Comment by msalahz** [comment]
Hi, I tracked the source of this warring in my app and I found two addons that coz it. 
1. ember-simple-auth, upgrading to last release fixed it.
2. ember-gestures, still using it to support ember versions < 2.3
